src/components/Menubar.js

```diff
@@ -1,11 +1,11 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import { Link } from 'react-router-dom'
-import { getInfo } from '../actions/info'
-import PropTypes from 'prop-types'
-import { Menu } from 'semantic-ui-react'
-import { dashboard, forms, submission } from '../urls'
-import menubar from './../styles/Menubar.css'
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getInfo } from '../actions/info';
+import PropTypes from 'prop-types';
+import { Menu } from 'semantic-ui-react';
+import { dashboard, forms, submission } from '../urls';
+// import menubar from './../styles/Menubar.css'
```

Comment:
same here

Reply:
No, I just commented out that code because it was never used in that file.